packages/api/src/controllers/room.ts

```diff
@@ -96,7 +253,7 @@ app.post("/webhook", express.raw({ type: "*/*" }), async (req, res) => {
     throw new BadRequestError(`no room name on event`);
   }
   const room = await db.room.get(roomId);
-  if (!room) {
+  if (!room || room.deleted) {
     throw new NotFoundError(`room not found`);
   }
```

Comment:
Hmm, in case of webhooks maybe we want to keep tracking info that we get after the room was deleted, so possibly not check the `deleted` field here. WDYT?

Reply:
Yeah that makes sense, and tbh I think if the room doesn't exist I'd want livekit to retry sending, from the testing I've done I don't think this will be a lot of failures but I'm just thinking of when the room starts up there will be a brief window when we may not have created the db item yet but it has been created on livekit's side.